eox_core/edxapp_wrapper/backends/enrollment_h_v1.py

```diff
@@ -38,13 +39,16 @@ def create_enrollment(*args, **kwargs):
             raise APIException('No user found with that email')
         else:
             username = match.username
-
     try:
         enrollment = api._data_api().create_course_enrollment(username, course_id, mode, is_active)
         if enrollment_attributes is not None:
             api.set_enrollment_attributes(username, course_id, enrollment_attributes)
-    except (UserNotFoundError, InvalidEnrollmentAttribute, CourseNotFoundError, CourseEnrollmentFullError, CourseEnrollmentClosedError, CourseEnrollmentExistsError) as e:
-        enrollment = None
+    except CourseEnrollmentExistsError as e:
+        if force_registration:
+            enrollment = api._data_api().update_course_enrollment(username, course_id, mode, is_active)
+        else:
+            raise APIException(repr(e) + ", use force_registration=1 if updating is what you need")
```

Comment:
"use force_registration to update the existing enrollment"

Reply:
Done